Update the readme (#86)
TheAlgorithms/PHP: README.md

> #FOR EDUCATIONAL PURPOSE:

Reviewer: ```suggestion
# The Algorithms - PHP
```

Reviewer: It's uncommon in the open-source world, at least from what I've seen, to have the first line in the README show something other than the title of the repo

> #The Algorithms - PHP

Reviewer: ```suggestion
```

> ### All algorithms implemented in Php

Reviewer: ```suggestion
### All algorithms implemented in Python - for education
```

I'd suggest following what's been done in the other `TheAlgorithms` repos, e.g. [Python](https://github.com/TheAlgorithms/Python/blob/master/README.md#all-algorithms-implemented-in-python---for-education)

> The Implementation on this repo are all exaqmples and may in anytime be less efficient than the implementations on the PHP standard Library.

Reviewer: ```suggestion
The Implementations in this repo are examples and may be less efficient than the implementations in the PHP standard Library.
```

## Contribution Guidelines:
To Contribute to this repo please take your time to read our
[Contribution Guidelines](CONTRIBUTING.md) before you contribute.

> ## Community Channel
> Let's connect and grow!
> [Gitter](https://gitter.im/TheAlgorithms)!
> [Twitter]
> [Discord]
> Please join us.

Reviewer: ```suggestion
## Community Channels

We're on [Discord](https://discord.gg/c7MnfGFGa6) and [Gitter](https://gitter.im/TheAlgorithms)! Community channels are great for you to ask questions and get help. Please join us!
```

I'd suggest following the [Python](https://github.com/TheAlgorithms/Python/blob/master/README.md#community-channels) (or another `TheAlgorithms` project).

## List of Algorithms

> See our [directory](DIRECTORY.md) to see the list of algorithms we have currently.

Reviewer: ```suggestion
See the [directory](DIRECTORY.md) for easier navigation and to view the current list of algorithms.
```
